Tutorials_UsingAPIs/Flow-Control-Busy-Wait/main.cpp

```diff
@@ -6,9 +6,9 @@ int main() {
     while(1) {
         myled = 1;
         // printf("LED On\r\n");
-        wait(0.2);
+        wait_us(200000);
```

Comment:
Same here, no need to busy wait.

Reply:
As per previous comment, I propose that this is the only example that uses `wait_us`. Let's emphasize further by adding a comment
```
wait_us(200000); // Busy wait
```

When updating the tutorial, we should add a forth option : "sleep" option.